force-app/main/default/triggers/Count.trigger

```diff
@@ -0,0 +1,70 @@
+trigger Count on Task (after insert, after update, after delete) {
+  
+    
+      Set<Id> accIds = new Set<Id>();
+        if (trigger.isInsert) {
+            for (Task task : Trigger.new){
+                if (task.whatId !=null && String.valueOf(task.whatId).startsWith('001')) {
```

Comment:
So what exactly happens with the accIds?

Reply:
and why are there three checks for each Trigger Event if the logic is exactly the same for all of them?